interp/notation_ops.ml

```diff
@@ -1344,7 +1361,7 @@ let rec match_ inner u alp metas sigma a1 a2 =
 
   (* Matching compositionally *)
   | GVar id1, NVar id2 when alpha_var id1 id2 (fst (snd alp)) -> sigma
-  | GRef (r1,_), NRef r2 when (GlobRef.equal r1 r2) -> sigma
+  | GRef (r1,u1), NRef (r2,u2) when (GlobRef.equal r1 r2) && compare_universe_instances false u1 u2 -> sigma (* TODO: sensibly match universe instances *)
```

Comment:
What does "sensibly" mean here?

Reply:
That I forgot to remove this comment when I added `compare_universe_instances` :wink:.